samples/HelloWorld/HelloWorld.al

```diff
@@ -11,6 +11,6 @@ codeunit 70051001 HelloWorld
     var
         HelloText : Codeunit GreetingsManagement;
     begin
-        Message(HelloText.GetRandomGreeting() + ', ' + Rec.Name);
+        Message('%1, %2',HelloText.GetRandomGreeting(),Rec.Name);
```

Comment:
A coding convention that we would like to follow for the samples (and perhaps for AL in general - we'd love to hear more feedback) is to use space after comma delimiters in argument lists.